.vscode/settings.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
Suggests adding some vs-specific filtering [in the `.gitignore` file](https://github.com/awslabs/aws-lc/blob/main/.gitignore).